test/puppetlabs/puppetdb/mq_test.clj

```diff
@@ -109,8 +110,34 @@
       (is (= size-bytes (.. broker (getSystemUsage) (getTempUsage) (getLimit)))))))
 
 (deftest test-jmx-enabled
-  (svc-utils/without-jmx
+  (without-jmx
    (svc-utils/with-puppetdb-instance
      (is (thrown-with-msg? clojure.lang.ExceptionInfo
                            #"status 404"
                            (svc-utils/current-queue-depth))))))
+
+(deftest queue-operations
+  (with-test-broker "test" conn
+    (remove-queue! "test" "foo")
+    (is (thrown+? [:type ::mq/queue-not-found] (queue-size "test" "foo")))
+    (send-message! conn "foo" "1")
+    (send-message! conn "bar" "1")
+    (bounded-drain-into-vec! conn "foo" 1)
```

Comment:
is the idea here to test that bounded-drain-into-vec successfully empties the queue? If so do we need another check above this to ensure that the message made it in to start with?